ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py

```diff
@@ -1271,3 +1271,65 @@ def test_tensorflow_linspace(
         axis=axis,
         on_device=on_device,
     )
+   
+
+# tile
+@st.composite
+# why do we name this starting with an underscore?
+# Is there a standard naming convention I can follow for function names?
+def _multiple_shape_helper(draw):
+    # generate dtypes_and_input
+
+    dtypes_and_input = draw(
+        helpers.dtype_and_values(
+            available_dtypes = helpers.get_dtypes("numeric")))
+    
+    #obtain the value of input dimensions from already existing ivy.shape function
+    input_dims = len(ivy.shape(input))
+    
+    shape = draw(
+        helpers.get_shape(
+            # as this is a 1-D tensor, I'm limited dimensions to 1
+            # the size, ie the length of the tensor will be the same as the no. of dimensions in the input
+            min_num_dims=1, max_num_dims=1, min_dim_size=1, max_dim_size=input_dims
```

Comment:
The length should always be equal to `input_dims`, so `min_dim_size=input_dims` as well.